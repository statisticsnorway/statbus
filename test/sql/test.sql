SELECT 1 AS test;
