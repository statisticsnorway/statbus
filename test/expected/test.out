SELECT 1 AS test;
 test 
------
    1
(1 row)

